template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Areas/IdentityService/Data/Migrations/00000000000001_SeedApplications.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Company.WebApplication1.Identity.Data;
+using JetBrains.Annotations;
+
+namespace Company.WebApplication1.Identity.Data.Migrations
+{
+    [DbContext(typeof(IdentityServiceDbContext))]
+    [Migration("00000000000001_SeedApplications")]
+    public class SeedApplications : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            SeedClientApplication.Seed(migrationBuilder);
```

Comment:
I'll get rid of the classes, but should there be two helper methods within the migration so that it's easier to see where the seed data is getting added?